jackson/src/main/java/com/spotify/hamcrest/jackson/IsJsonBoolean.java

```diff
@@ -58,14 +58,14 @@ protected boolean matchesNode(BooleanNode node, Description mismatchDescription)
     if (booleanMatcher.matches(value)) {
       return true;
     } else {
-      mismatchDescription.appendText("was a boolean node ");
+      mismatchDescription.appendText("was a boolean node with value that ");
       booleanMatcher.describeMismatch(value, mismatchDescription);
       return false;
     }
   }
 
   @Override
   public void describeTo(Description description) {
-    description.appendText("a boolean node ").appendDescriptionOf(booleanMatcher);
+    description.appendText("a boolean node with value that ").appendDescriptionOf(booleanMatcher);
```

Comment:
Seems like you should change the no-arg version of all these public functions then, i.e. make jsonBoolean() in this class call jsonBoolean(is(anything())) rather than jsonBoolean(anything())